test/test-results/10893_STS.md

```diff
@@ -0,0 +1,389 @@
+>>> | 2 |Check the displayed page number equals 1  | Page number = 1 |  |
+>>> | 3 | Enter page 4 on the pagination control and select ENTER | Pagination control resets to Page 1 and correct projects are displayed |  |
+>>> | 4 | Enter page 2 on the pagination control and select ENTER | Page 2 is loaded with correct projects |  |
+>>> | 5 | Enter page 4 on the pagination control and select ENTER | Pagination control resets to Page 2 and correct projects are displayed |  |
+>>> | 6 | Enter page 3 on the pagination control and select ENTER | Page 3 is loaded with correct projects |  |
+>>> | 7 | Enter page 4 on the pagination control and select ENTER | Pagination control resets to Page 3 and correct projects are displayed |  |
+>>> | 8 | Enter page 0 on the pagination control and select ENTER | Pagination control resets to Page 3 and correct projects are displayed |  |
+>>> | 9 | Enter page -1 on the pagination control and select ENTER | Pagination control resets to Page 3 and correct projects are displayed |  |
+>>> | 10 | Enter page 100 on the pagination control and select ENTER | Pagination control resets to Page 3 and correct projects are displayed |  |
+>>> | 11 | Enter page abc on the pagination control and select ENTER | Pagination control resets to Page 3 and correct projects are displayed |  |
+>>> | 12 | Enter page 1a on the pagination control and select ENTER | Pagination control resets to Page 3 and correct projects are displayed |  |
+---
+>>>###### **TC2.1.3.3.9 Selecting 'My projects' when on the last page of all projects, sends user to correct last page if there are fewer pages for their projects**
+>>> | **JIRA ID(s):** | [NERCDL-599](https://jira.ceh.ac.uk/browse/NERCDL-599?jql=key%20in%20(NERCDL-599)), [NERCDL-626](https://jira.ceh.ac.uk/browse/NERCDL-626?jql=key%20in%20(NERCDL-626)) |OVERALL RESULT: | |
+>>> | ------------ | --------- | --------- | ------|
+>>> | **Number** | **Step** | **Expected Result** | **Result** |
+>>> | 1 |Ensure 30 projects have been created by the logged in user and the user has access to to a further 1 project not created by them | 30 current user projects and 1 other user project are available |  |
+>>> | 2 | Ensure that My Projects is deselected | Option is deselected, all projects the user can view are displayed in pages of 10 each or less |  |
+>>> | 3 | Select the last page | Last page (should be Page 4) is selected. correct projects are displayed |  |
+>>> | 4 | Select My Projects option | Option is selected, only the current user's projects are displayed, last page is displayed (should be page 3) |  |
+---
+>##### 2.1.4 Open Project
+
+### 2.2 Project View
+
+>##### 2.2.1 Select Project
+
+>##### 2.2.2 Information
+
+>##### 2.2.3 Analysis
+>>##### 2.2.3.1 Notebooks
+>>>###### **T2.2.3.1 User can edit existing Notebooks**
+>>> | **JIRA ID(s):** | [NERCDL-613](https://jira.ceh.ac.uk/browse/NERCDL-613?jql=key%20in%20(NERCDL-613)) |OVERALL RESULT: | PASS |

```

Comment:
Should the results be in the STS?

Reply:
Sorry, just realised my mistake - I saw 'STS' and thought it was the script, not the results!